gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -458,6 +462,26 @@ def main(args):
                 cochran_mantel_haenszel_test=res.cmh_ht.ht()[
                     ht.key
                 ].cochran_mantel_haenszel_test,
+                join_metric_data_type=hl.case()
+                .when(
+                    (hl.is_defined(ht.genomes_grpmax.AC))
+                    & hl.is_defined(ht.exomes_grpmax.gnomad.AC),
+                    "both",
+                )
+                .when(hl.is_defined(ht.genomes_grpmax.AC), "genomes")
+                .when(hl.is_defined(ht.exomes_grpmax.gnomad.AC), "exomes")
+                .default(hl.missing(hl.tstr)),
```

Comment:
There are a lot of NA already, even in my test gene. A lot of grpmax.AC is missing.

Reply:
`grpmax` is missing for any variant exclusive to bottleneck pop.

Reply:
It will be part of the existing `missingness` checks which just return the percent of missing values for an annotation. @ch-kr, are you suggesting instead of defaulting to missing, have a when statement that checks and then assigns/keeps missingness? and then the last bit would be `or_error`?

Reply:
nope, just wanted to double check why there was an explicitly set missing case. makes sense to keep this as is given this will already be checked with the rest of the missingness checks (so we should know if there is an error)